frontend/js/index.js

```diff
@@ -24,9 +24,11 @@ function accountLogin (email, username) {
           .then(account => {
             game.account = account;
             displayGame();
+            leaderboard.fetchUsers();
           });
       } else {
         displayGame();
+        leaderboard.fetchUsers();
```

Comment:
Move outside of the if-else chain, if you need the account set it can be put after.